examples/test-controls/index.html

```diff
@@ -11,7 +11,7 @@
     <a-scene stats="true">
       <!-- We wrap the camera in an <a-entity> so we can position it without colliding with the HMD updates -->
       <a-entity>
-        <a-entity position="0 1.8 4" camera look-controls wasd-controls></a-entity>
+        <a-entity position="0 1.8 4" camera controls></a-entity>
```

Comment:
I think it is good to leave to `controls` to be agnostic enough to be used for anything that takes input, and then modifies position/rotation of any entity. This example just displays using the out of the box default control scheme, so it won't show much info (until it injects all of its components).